cmd/abc/appbase_import.go

```diff
@@ -341,11 +341,32 @@ func verifyConnections(adaptors map[string]adaptor.Adaptor) error {
 }
 
 func verifyConnectionsWithoutDestination(srcConfig map[string]interface{}) error {
+
```

Comment:
You could remove these custom checks (added in commit 33043c4) now.
It's taken care of by the individual adaptor's Verify methods.

Reply:
Done